catalog/language.go

```diff
@@ -0,0 +1,8 @@
+package catalog
+type Language struct {
```

Comment:
Do we have an agreement on code documentation for `go` language? If no - then let's agree on that 😉!